extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb

```diff
@@ -0,0 +1,431 @@
+                "\n",
+                "\n",
+                "Steps of this procedure include:\n",
+                "1. Connect to Azure subscription\n",
+                "1. Provision resource group for SQL VM migration\n",
+                "2. Create a storage account\n",
+                "3. Configure Network Settings\n",
+                "3. Provision SQL VM\n",
+                "4. Configure SQL VM IaaS agent"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "e479b550-d6bd-49c5-965a-34a7d1d16412"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Notebook setup"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "6251b830-476f-48f2-af1e-3680b541e455"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "import pandas, sys, os, json, html, getpass, time, tempfile, string, random, subprocess\r\n",
+                "\r\n",
+                "def run_command(command, json_decode = True):\r\n",
+                "    print(command)\r\n",
+                "    process = subprocess.Popen(command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)\r\n",
+                "    output, error = process.communicate()\r\n",
+                "    if process.returncode != 0: \r\n",
+                "        print(\"Process failed %d \\n%s\" % (process.returncode, error.decode(\"utf-8\")))\r\n",
+                "        raise Exception()\r\n",
+                "    if output:\r\n",
+                "        output = output.decode(\"utf-8\")\r\n",
+                "        print(output)\r\n",
+                "        try:\r\n",
+                "            return json.loads(output)\r\n",
+                "        except:\r\n",
+                "            return output\r\n",
+                "\r\n",
+                "def get_random_string(length):\r\n",
+                "    letters = string.ascii_lowercase\r\n",
+                "    result_str = ''.join(random.choice(letters) for i in range(length))\r\n",
+                "    print(\"Random string of length\", length, \"is:\", result_str)\r\n",
+                "    return result_str"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "dd0caed8-b12e-4a9e-9705-fdcf5fa3ff7e"
+            },
+            "outputs": [],
+            "execution_count": null
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Set variables"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "37db2e50-dcde-4dd5-820c-7dc11212f1eb"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "if \"AZDATA_NB_VAR_AZURE_SQLVM_PASSWORD\" in os.environ:\r\n",
```

Comment:
Given that this notebook isn't using azdata at all I'd suggest changing the var names so there isn't any confusion there. I seem to recall the deployment wizard using the prefix to determine what vars to insert into the doc though so you might need to also change the deployment extension code. @alanrenmsft